src/app/services/keycloak.service.ts

```diff
@@ -53,7 +66,7 @@ export class KeycloakService {
         };
 
         this.keycloakAuth.onAuthLogout = () => {
-          console.log('onAuthLogout');
+          // console.log('onAuthLogout');
```

Comment:
Does this need to be commented out?

Reply:
I believe so since it is commented out for Attendance & Revenue, and it also violates a common linting rule.
The other logging calls in this file use `this.logger.log()`